src/_ert_forward_model_runner/reporting/event.py

```diff
@@ -74,9 +78,9 @@ def __init__(self, evaluator_url, token=None, cert_path=None):
 
         self._ens_id = None
         self._real_id = None
-        self._event_queue = queue.Queue()
+        self._event_queue: queue.Queue[events.Event] | None = queue.Queue()
```

Comment:
Why `None`?

Reply:
It is supposed to be inside the Queue type. thanks :)